sqlglot/expressions.py

```diff
@@ -5985,7 +5985,7 @@ class TimeStrToDate(Func):
 
 
 class TimeStrToTime(Func):
-    pass
+    arg_types = {"this": True, "timezone": False}
```

Comment:
Ironically, I picked `timezone` because that's what `TimeToStr` was using :)

I'll change it and I guess a breaking change to `TimeToStr` isnt a problem?

Reply:
it's not if we tag it with !